Library/Formula/mydumper.rb

```diff
@@ -0,0 +1,118 @@
+class Mydumper < Formula
+  desc "How MySQL DBA & support engineer would imagine 'mysqldump' ;-)"
+  homepage "https://launchpad.net/mydumper"
+  url "https://launchpad.net/mydumper/0.6/0.6.2/+download/mydumper-0.6.2.tar.gz"
+  sha256 "fa28563e8967752828954c5d81e26ef50aad9083d50a977bf5733833b23e3330"
+
+  option "without-docs", "Don't build man pages"
+
+  depends_on "cmake" => :build
+  depends_on :python => :build if MacOS.version <= :snow_leopard && build.with?("docs")
+  depends_on "glib"
+  depends_on "mysql"
+  depends_on "pcre"
+  depends_on "pkg-config" => :build
```

Comment:
World's smallest nit: please group this dep with the `:build` deps further up (Although Python can stop being a dep since we declare it in the `sphinx-doc` formula I referenced below).